src/index.ts

```diff
@@ -103,7 +114,7 @@ app.use(keycloak.middleware({ logout: '/logout' }));
 
 // Put the login route before the expired redirect to prevent an
 // endless loop.
-app.get('/login', keycloak.protect(), (req: GrantedRequest, res: Response) => {
+app.get('/login', [keycloak.protect(), updateOndernemerEmailMiddleware], (req: GrantedRequest, res: Response) => {
     if (req.query.next) {
```

Comment:
Volgens mij zijn die `[ ]` haakjes niet nodig bij middlewares, maar weet het niet zeker.
`app.get('/login', keycloak.protect(), updateOndernemerEmailMiddleware, (req: GrantedRequest, res: Response) => {...})`

Reply:
zonder list lijkt inderdaat ook te werken. in docs staat het als list aangegeven, maar dat is wss omdat het optional is.
https://expressjs.com/en/4x/api.html#router.METHOD